Ombi.UI/Modules/SearchModule.cs

```diff
@@ -222,6 +229,36 @@ public class SearchModule : BaseAuthModule
             return await ProcessMovies(MovieSearchType.Search, searchTerm);
         }
 
+        private async Task<Response> SearchActor(string searchTerm)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<Response> SearchActor(string searchTerm, bool filterExisting)
+        {
+            var movies = TransformMovieListToMovieResultList(await MovieApi.SearchActor(searchTerm, AlreadyAvailable).ConfigureAwait(false));
+            return await TransformMovieResultsToResponse(movies);
+        }
+
+        private async Task<bool> AlreadyAvailable(int id, string title, string year)
+        {
+            await Task.Yield();
+            return IsMovieInCache(id, String.Empty) || PlexChecker.IsMovieAvailable(plexMovies(), title, year);
+        }
+
+        private IEnumerable<PlexContent> plexMovies()
```

Comment:
My goal here was to cache the results from the Plex repository as it seemed it was getting called a lot.  Maybe that's a necessary evil?

In any event, with my new checkbox for "only show new results," I wanted a subsequent search to filter out items that were recently requested.  That isn't working, so maybe I'm going about it in the wrong way here.

Any thoughts, @tidusjar?